webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -56,98 +51,97 @@ const AuditCard = ({ hostName }) => {
-                    >
-                      <span>{`${audit.action} (by ${audit.user_name})`}</span>
-                    </AccordionToggle>
-                    <AccordionContent
-                      isHidden={
-                        ativeAccordion !== `${audit.request_uuid}-${index}`
-                      }
+                      isCompact
+                      aria-label="Audits"
                     >
-                      <DataList
-                        style={{
-                          width: '350px',
-                          overflowX: 'hidden',
-                        }}
-                        isCompact
-                        aria-label="Audits"
-                      >
-                        {audit &&
-                          Object.entries(audit.audited_changes).map(
-                            ([key, value], i) => (
-                              <DataListItem aria-labelledby="">
-                                <DataListItemRow>
-                                  <DataListItemCells
-                                    dataListCells={[
-                                      <DataListCell key={key}>
-                                        <span> {key}</span>
-                                      </DataListCell>,
-                                      <DataListCell
-                                        key={`old-value-${i}`}
-                                        style={{
-                                          width: '350px',
-                                          overflowX: 'hidden',
-                                        }}
-                                      >
-                                        {value && (
-                                          <ElipsisWithTooltip>
-                                            <mark
-                                              style={{
-                                                backgroundColor: '#FFC0CB',
-                                              }}
-                                            >
-                                              {value[0]}
-                                            </mark>
-                                          </ElipsisWithTooltip>
-                                        )}
-                                      </DataListCell>,
-                                      <DataListCell
-                                        style={{
-                                          width: '350px',
-                                          overflowX: 'hidden',
-                                        }}
-                                        key={`new-value-${i}`}
-                                      >
-                                        {value && (
+                      {audit &&
+                        Object.entries(audit.audited_changes).map(
+                          ([key, value], i) => (
+                            <DataListItem aria-labelledby="">
```

Comment:
Not directly related to this PR, but I'm seeing an error about this component missing a key